cmd/flux/bootstrap_git.go

```diff
@@ -164,7 +164,6 @@ func bootstrapGitCmdRun(cmd *cobra.Command, args []string) error {
 		// Configure repository URL to match auth config for sync.
 		repositoryURL.User = nil
 		repositoryURL.Scheme = "https"
-		repositoryURL.Host = repositoryURL.Hostname()
```

Comment:
Thanks for your patience on this. One last thing to address @hiddeco comment
>> Given the scenario where someone configures a --url ssh://git@example.com:9999/repository.git to perform Git operations from their machine over SSH, but also sets --token-auth to enable HTTPS on the cluster side, it will now result in the SSH port being configured for the HTTPS URL.

I don't think that it's a common use case, but to preserve behavior, can you keep `repositoryURL.Host = repositoryURL.Hostname()` for non http schemes?

Reply:
Sorry for being unclear. 

```go
if bootstrapArgs.tokenAuth {
  ...
  repositoryURL.Host = repositoryURL.Hostname()
}
```

to 
```go
if bootstrapArgs.tokenAuth {
  ...
  repositoryURL.Host = repositoryURL.Hostname()
    if repositoryURL.Scheme != "https" && repositoryURL.Scheme != "http" {
      repositoryURL.Host = repositoryURL.Hostname()
    } 
```